### DISCUSSION TITLE
Is it possible to put the 'hats' under the characters?

### DISCUSSION THREAD

**Discussion by autosquash** [cursorless-dev/cursorless#1098] [Ideas]
I find it would be more readable for me to place the hats under the characters. Is it possible? Currently, using `hatVerticalOffset `setting, the hats disappear before reaching the lowest point of the character. I don't know if it is intended.

**Comment by pokey** [comment]
It should be doable, keeping in mind that underlining is a fairly common annotation, so you may find the hats commonly fighting with an underline. Feel free to file an issue, and if you want to take a crack at it I can point you to the right place in the code

**Reply by auscompgeek** [reply]
There's a `Create issue from discussion` button, should we press it? 😛

![image](https://user-images.githubusercontent.com/128854/198838726-f1c5b9ca-746f-4d77-af12-2a1b03229f52.png)

**Reply by autosquash** [reply]
> Feel free to file an issue

Ok, I just created [the issue](https://github.com/cursorless-dev/cursorless/issues/1122)


> if you want to take a crack at it I can point you to the right place in the code

Yes please, I don't know if I can do it but I can try.

**Comment by auscompgeek** [comment]
Given we've agreed allowing this is reasonable, we can continue discussion in the linked issue #1122. Closing this thread :)
